### PR TITLE
SNAP-141 Configure acceptance tests to run with ie edge and safari for cross browser testing

### PR DESCRIPTION
**Pull request by NagaPradeepGajula-osi**
### Jira Story

- [Gather Browser Testing requirements](https://osi-cwds.atlassian.net/browse/SNAP-141)

Documentation for the setup is done here https://github.com/ca-cwds/acceptance_testing/wiki/Cross-Browser-Testing-Setup

### CODE REVIEW

File: helpers/form_helper.rb [hunk 1]
```diff
@@ -23,10 +23,10 @@ def select_fields(**args)
     end
   end
 
-  def fill_input_filds(**args)
-    self::INPUT_FIELDS.each do |key, value|
-      clear_field(value)
-      fill_in(value, with: args[key])
+  def fill_input_fields(args = {})
+    self.class::INPUT_FIELDS.each do |key, value|
```
Comment (money) [inline helpers/form_helper.rb]: remove `.class` from `self.class`

**Comment by money** [comment]
Looks good. Just one change.

**Comment by NagaPradeepGajula-osi** [comment]
retest this please

**Comment by NagaPradeepGajula-osi** [comment]
retest this please

**Comment by NagaPradeepGajula-osi** [comment]
retest this please

**Comment by NagaPradeepGajula-osi** [comment]
retest this please

**Comment by NagaPradeepGajula-osi** [comment]
retest this please

**Pull request merged**